admin_guide/scheduler.adoc

```diff
@@ -367,3 +367,204 @@ Since the scheduler is a plug-in, it can be replaced in favor of an alternate
 implementation. The scheduler code has a clean separation that watches new pods
 as they get created and identifies the most suitable node to host them. It then
 creates bindings (pod to node bindings) for the pods using the master API.
+
+
+[[controlling-pod-placement]]
+== Controlling Pod Placement
+
+As a cluster administrator, you can set a policy to prevent application
+developers with certain roles from targeting specific nodes when scheduling
+pods. 
+
+[IMPORTANT]
+====
+This process involves the *pod/binding* permission
+link:../admin_guide/manage_authorization_policy.html[role], which is needed to
+target particular nodes. The constraint on the use of the `nodeSelector` field
+of a pod configuration is based on the *pods/binding* permission and the
+`nodeSelectorLabelBlacklist` configuration option.
+====
+
+The `nodeSelectorLabelBlacklist` field of a master configuration file gives you
+control over the labels that certain roles can specify in new
```

Comment:
looking at e.g. https://docs.openshift.com/enterprise/3.0/admin_guide/scheduler.html#configurable-predicates it seems like this could be rewritten like "...certain roles can specify in a pod's `nodeSelector` field."